syncer/syncer_test.go

```diff
@@ -141,14 +141,16 @@ func (s *testSyncerSuite) SetUpSuite(c *C) {
 		Dir: loaderDir,
 	}
 	s.cfg = &config.SubTaskConfig{
-		From:         config.GetDBConfigFromEnv(),
-		To:           config.GetDBConfigFromEnv(),
-		ServerID:     101,
-		MetaSchema:   "test",
-		Name:         "syncer_ut",
-		Mode:         config.ModeIncrement,
-		Flavor:       "mysql",
-		LoaderConfig: loaderCfg,
+		From:             config.GetDBConfigFromEnv(),
+		To:               config.GetDBConfigFromEnv(),
+		ServerID:         101,
+		MetaSchema:       "test",
+		Name:             "syncer_ut",
+		ShadowTableRules: []string{"^_(.+)_(?:new|gho)$"},
```

Comment:
no need to manually add these two rules here, subtask.Adjust already do this

Reply:
We didn't adjust the whole subtask config here.